understanding/seizures-and-physical-reactions.html

```diff
@@ -54,59 +52,45 @@
       <div class="default-grid with-gap leftcol">
          <aside class="box nav-hack sidebar standalone-resource__sidebar ">
             <nav>
-               <header class="box-h ">Page Contents</header>
+               <header class="box-h ">このページの内容</header>
                <div class="box-i">
                   <ul>
-                     <li><a href="#intent">Intent</a></li>
-                     <li><a href="#success-criteria">Success Criteria</a></li>
+                     <li><a href="#intent">意図</a></li>
+                     <li><a href="#success-criteria">達成基準</a></li>
                   </ul>
                </div>
             </nav>
          </aside>
          <main id="main" class="standalone-resource__main">
 
-         <!-- 日本語訳されるまでの一時的な注記 -->
-         <div class="untranslated-note"><p>このWCAG 2.2 解説書の日本語訳は作業中となっています。WCAG 2.1 解説書の日本語訳をご利用いただけます。<br>[ <a href="https://waic.jp/translations/WCAG21/Understanding/">WCAG 2.1 解説書</a> ]</p></div>
-            <h1><span class="standalone-resource__type-of-guidance">Understanding 
-                  				Guideline 2.3:</span>Seizures and Physical Reactions
+            <h1><span class="standalone-resource__type-of-guidance">解説書 
+                  				ガイドライン 2.3:</span>発作と身体的反応
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Guideline </header>
+               <header class="box-h  box-h-icon">ガイドライン</header>
                <div class="box-i">
-                  <p>Do not design content in a way that is known to cause seizures or physical reactions.</p>
+                  <p>発作や身体的反応を引き起こすようなコンテンツを設計しないこと。</p>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section xmlns:wcag="https://www.w3.org/WAI/GL/" id="intent">
-               <h2>Intent</h2>
-               <p>Some people with seizure disorders can have a seizure triggered by flashing visual
-                  content. Most people are unaware that they have this disorder until it strikes. In
-                  1997, a cartoon on television in Japan sent over 700 children to the hospital, including
-                  about 500 who had seizures. Warnings do not work well because they are often  missed,
-                  especially by children
-                  who may in fact not be able to read them.
+               <h2>意図</h2>
+               <p>発作性障害のある利用者は、閃光を放つ視覚的なコンテンツによって発作を引き起こすことがある。ほとんどの利用者は、発作に襲われるまでは自分がこの障害を持っていることに気づかない。1997 年に、日本でテレビのアニメ番組が 700 人以上の子どもたちを病院に搬送させる事態を招き、そのうち約 500 人が発作を引き起こした。特に、警告を実際に読むことができない子どもの利用者に警告はよく見逃されるので、その警告はあまり効果がない。
```

Comment:
問題ないのですが、末尾の文は「その警告」が何を指しているのかわかりにくい感じがします。こんな感じにしてはどうでしょうか。

```suggestion
               <p>発作性障害のある利用者は、閃光を放つ視覚的なコンテンツによって発作を引き起こすことがある。ほとんどの利用者は、発作に襲われるまでは自分がこの障害を持っていることに気づかない。1997 年に、日本でテレビのアニメ番組が 700 人以上の子どもたちを病院に搬送させる事態を招き、そのうち約 500 人が発作を引き起こした。警告は、よく見逃される上に、特に子どもの利用者はそれを実際に読めないこともあるため、あまり効果がない。
```